.ci_support/migrations/cuda120.yaml

```diff
@@ -2,10 +2,6 @@ migrator_ts: 1682985063
 __migrator:
   kind:
     version
-  # Vendor CUDA 12 migrator to use GCC 11.
-  # This is needed to workaround a pybind + GCC 12 + nvcc 12 bug
-  # xref: https://github.com/pybind/pybind11/issues/4606
-  use_local: true
```

Comment:
The issue referenced above was fixed in pybind11's 2.12.0 with this change: https://github.com/pybind/pybind11/commit/3414c56b6c7c521d868c9a137ca2ace2e26b5b2e

We included pybind11 2.12.0 in conda-forge some time ago: https://github.com/conda-forge/pybind11-feedstock/pull/94

Given this have dropped this `use_local: true` (to devendor this migrator) and refreshed it to match [the global migrator]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/0f05ff70a3838e369ef9367f5715de1713fb4c99/recipe/migrations/cuda120.yaml )

Also [re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks ) to update this feedstock